Build failure: sc-im

### Steps To Reproduce

Steps to reproduce the behavior:
1. build sc-im on nixpkgs-24.05-darwin

### Build log

```
nix-shell -p sc-im
this derivation will be built:
  /nix/store/pinl7arfp0w0d5kq48xdsvsz1la8d1v7-sc-im-0.8.3.drv
these 7 paths will be fetched (2.34 MiB download, 10.18 MiB unpacked):
  /nix/store/5qisdh152ljbjbx7h1gi60dxk92c6za0-die-hook
  /nix/store/b8nv992vghbnkca6acx9dl5v3x2iwbvk-gnuplot-6.0.0
  /nix/store/645j1xqiza8nnb71dbzlv751y7a15k53-libcerf-2.4
  /nix/store/7xr1dnrw1r26dcpwgifqdzqkh80pzla6-libzip-1.10.1
  /nix/store/83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev
  /nix/store/fjj2nd0c7xbbprn86drm6wlkgmccfi9q-make-shell-wrapper-hook
  /nix/store/fr75ka7wxv93nfpmmsnjk768y9pqdq2i-source
copying path '/nix/store/fr75ka7wxv93nfpmmsnjk768y9pqdq2i-source' from 'https://cache.nixos.org'...
copying path '/nix/store/5qisdh152ljbjbx7h1gi60dxk92c6za0-die-hook' from 'https://cache.nixos.org'...
copying path '/nix/store/645j1xqiza8nnb71dbzlv751y7a15k53-libcerf-2.4' from 'https://cache.nixos.org'...
copying path '/nix/store/7xr1dnrw1r26dcpwgifqdzqkh80pzla6-libzip-1.10.1' from 'https://cache.nixos.org'...
copying path '/nix/store/fjj2nd0c7xbbprn86drm6wlkgmccfi9q-make-shell-wrapper-hook' from 'https://cache.nixos.org'...
copying path '/nix/store/83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev' from 'https://cache.nixos.org'...
copying path '/nix/store/b8nv992vghbnkca6acx9dl5v3x2iwbvk-gnuplot-6.0.0' from 'https://cache.nixos.org'...
building '/nix/store/pinl7arfp0w0d5kq48xdsvsz1la8d1v7-sc-im-0.8.3.drv'...
Running phase: unpackPhase
unpacking source archive /nix/store/fr75ka7wxv93nfpmmsnjk768y9pqdq2i-source
source root is source/src
Running phase: patchPhase
Running phase: configurePhase
no configure script, doing nothing
Running phase: buildPhase
build flags: SHELL=/nix/store/8prdfb52i3sy1wf64w03apn481585i4f-bash-5.2p26/bin/bash prefix=/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3
yacc -d gram.y
mv y.tab.c gram.c
sed -f eres.sed < gram.y > experres.h
sed -f sres.sed < gram.y > statres.h
clang -Wall -g -DNCURSES -D_XOPEN_SOURCE_EXTENDED -D_GNU_SOURCE -DSNAME=\"sc-im\" -DHELP_PATH=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/sc-im\" -DLIBDIR=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/doc/sc-im\" -DDFLT_PAGER=\"less
\" -DDFLT_EDITOR=\"vim\" -DUSECOLORS -DHISTORY_FILE=\"sc-iminfo\" -DHISTORY_DIR=\".cache\" -DCONFIG_FILE=\"scimrc\" -DCONFIG_DIR=\".config/sc-im\" -DINS_HISTORY_FILE=\"sc-iminfo\" -DUNDO -DMAXROWS=65536 -DUSELOCALE -DMOUSE -DDEFAULT_COPY_TO_CLIPBOARD_CMD=\""xclip -i -se
lection clipboard <"\" -DDEFAULT_PASTE_FROM_CLIPBOARD_CMD=\""xclip -o -selection clipboard"\" -DDEFAULT_OPEN_FILE_UNDER_CURSOR_CMD=\""scopen"\" -DAUTOBACKUP -DHAVE_PTHREAD -DGNUPLOT -DODS -DXLSX -I/nix/store/8pznqjvlvmqll53cvmk9vmv6r45xp1wx-libxml2-2.12.7-dev/include/li
bxml2 -I/nix/store/83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/../83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/include   -c -o block.o block.c
clang -Wall -g -DNCURSES -D_XOPEN_SOURCE_EXTENDED -D_GNU_SOURCE -DSNAME=\"sc-im\" -DHELP_PATH=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/sc-im\" -DLIBDIR=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/doc/sc-im\" -DDFLT_PAGER=\"less
\" -DDFLT_EDITOR=\"vim\" -DUSECOLORS -DHISTORY_FILE=\"sc-iminfo\" -DHISTORY_DIR=\".cache\" -DCONFIG_FILE=\"scimrc\" -DCONFIG_DIR=\".config/sc-im\" -DINS_HISTORY_FILE=\"sc-iminfo\" -DUNDO -DMAXROWS=65536 -DUSELOCALE -DMOUSE -DDEFAULT_COPY_TO_CLIPBOARD_CMD=\""xclip -i -se
lection clipboard <"\" -DDEFAULT_PASTE_FROM_CLIPBOARD_CMD=\""xclip -o -selection clipboard"\" -DDEFAULT_OPEN_FILE_UNDER_CURSOR_CMD=\""scopen"\" -DAUTOBACKUP -DHAVE_PTHREAD -DGNUPLOT -DODS -DXLSX -I/nix/store/8pznqjvlvmqll53cvmk9vmv6r45xp1wx-libxml2-2.12.7-dev/include/li
bxml2 -I/nix/store/83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/../83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/include   -c -o buffer.o buffer.c
clang -Wall -g -DNCURSES -D_XOPEN_SOURCE_EXTENDED -D_GNU_SOURCE -DSNAME=\"sc-im\" -DHELP_PATH=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/sc-im\" -DLIBDIR=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/doc/sc-im\" -DDFLT_PAGER=\"less
\" -DDFLT_EDITOR=\"vim\" -DUSECOLORS -DHISTORY_FILE=\"sc-iminfo\" -DHISTORY_DIR=\".cache\" -DCONFIG_FILE=\"scimrc\" -DCONFIG_DIR=\".config/sc-im\" -DINS_HISTORY_FILE=\"sc-iminfo\" -DUNDO -DMAXROWS=65536 -DUSELOCALE -DMOUSE -DDEFAULT_COPY_TO_CLIPBOARD_CMD=\""xclip -i -se
lection clipboard <"\" -DDEFAULT_PASTE_FROM_CLIPBOARD_CMD=\""xclip -o -selection clipboard"\" -DDEFAULT_OPEN_FILE_UNDER_CURSOR_CMD=\""scopen"\" -DAUTOBACKUP -DHAVE_PTHREAD -DGNUPLOT -DODS -DXLSX -I/nix/store/8pznqjvlvmqll53cvmk9vmv6r45xp1wx-libxml2-2.12.7-dev/include/li
bxml2 -I/nix/store/83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/../83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/include   -c -o clipboard.o clipboard.c
clang -Wall -g -DNCURSES -D_XOPEN_SOURCE_EXTENDED -D_GNU_SOURCE -DSNAME=\"sc-im\" -DHELP_PATH=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/sc-im\" -DLIBDIR=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/doc/sc-im\" -DDFLT_PAGER=\"less
\" -DDFLT_EDITOR=\"vim\" -DUSECOLORS -DHISTORY_FILE=\"sc-iminfo\" -DHISTORY_DIR=\".cache\" -DCONFIG_FILE=\"scimrc\" -DCONFIG_DIR=\".config/sc-im\" -DINS_HISTORY_FILE=\"sc-iminfo\" -DUNDO -DMAXROWS=65536 -DUSELOCALE -DMOUSE -DDEFAULT_COPY_TO_CLIPBOARD_CMD=\""xclip -i -se
lection clipboard <"\" -DDEFAULT_PASTE_FROM_CLIPBOARD_CMD=\""xclip -o -selection clipboard"\" -DDEFAULT_OPEN_FILE_UNDER_CURSOR_CMD=\""scopen"\" -DAUTOBACKUP -DHAVE_PTHREAD -DGNUPLOT -DODS -DXLSX -I/nix/store/8pznqjvlvmqll53cvmk9vmv6r45xp1wx-libxml2-2.12.7-dev/include/li
bxml2 -I/nix/store/83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/../83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/include   -c -o color.o color.c
clang -Wall -g -DNCURSES -D_XOPEN_SOURCE_EXTENDED -D_GNU_SOURCE -DSNAME=\"sc-im\" -DHELP_PATH=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/sc-im\" -DLIBDIR=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/doc/sc-im\" -DDFLT_PAGER=\"less
\" -DDFLT_EDITOR=\"vim\" -DUSECOLORS -DHISTORY_FILE=\"sc-iminfo\" -DHISTORY_DIR=\".cache\" -DCONFIG_FILE=\"scimrc\" -DCONFIG_DIR=\".config/sc-im\" -DINS_HISTORY_FILE=\"sc-iminfo\" -DUNDO -DMAXROWS=65536 -DUSELOCALE -DMOUSE -DDEFAULT_COPY_TO_CLIPBOARD_CMD=\""xclip -i -se
lection clipboard <"\" -DDEFAULT_PASTE_FROM_CLIPBOARD_CMD=\""xclip -o -selection clipboard"\" -DDEFAULT_OPEN_FILE_UNDER_CURSOR_CMD=\""scopen"\" -DAUTOBACKUP -DHAVE_PTHREAD -DGNUPLOT -DODS -DXLSX -I/nix/store/8pznqjvlvmqll53cvmk9vmv6r45xp1wx-libxml2-2.12.7-dev/include/li
bxml2 -I/nix/store/83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/../83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/include   -c -o conf.o conf.c
clang -Wall -g -DNCURSES -D_XOPEN_SOURCE_EXTENDED -D_GNU_SOURCE -DSNAME=\"sc-im\" -DHELP_PATH=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/sc-im\" -DLIBDIR=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/doc/sc-im\" -DDFLT_PAGER=\"less
\" -DDFLT_EDITOR=\"vim\" -DUSECOLORS -DHISTORY_FILE=\"sc-iminfo\" -DHISTORY_DIR=\".cache\" -DCONFIG_FILE=\"scimrc\" -DCONFIG_DIR=\".config/sc-im\" -DINS_HISTORY_FILE=\"sc-iminfo\" -DUNDO -DMAXROWS=65536 -DUSELOCALE -DMOUSE -DDEFAULT_COPY_TO_CLIPBOARD_CMD=\""xclip -i -se
lection clipboard <"\" -DDEFAULT_PASTE_FROM_CLIPBOARD_CMD=\""xclip -o -selection clipboard"\" -DDEFAULT_OPEN_FILE_UNDER_CURSOR_CMD=\""scopen"\" -DAUTOBACKUP -DHAVE_PTHREAD -DGNUPLOT -DODS -DXLSX -I/nix/store/8pznqjvlvmqll53cvmk9vmv6r45xp1wx-libxml2-2.12.7-dev/include/li
bxml2 -I/nix/store/83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/../83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/include   -c -o digraphs.o digraphs.c
clang -Wall -g -DNCURSES -D_XOPEN_SOURCE_EXTENDED -D_GNU_SOURCE -DSNAME=\"sc-im\" -DHELP_PATH=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/sc-im\" -DLIBDIR=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/doc/sc-im\" -DDFLT_PAGER=\"less\" -DDFLT_EDITOR=\"vim\" -DUSECOLORS -DHISTORY_FILE=\"sc-iminfo\" -DHISTORY_DIR=\".cache\" -DCONFIG_FILE=\"scimrc\" -DCONFIG_DIR=\".config/sc-im\" -DINS_HISTORY_FILE=\"sc-iminfo\" -DUNDO -DMAXROWS=65536 -DUSELOCALE -DMOUSE -DDEFAULT_COPY_TO_CLIPBOARD_CMD=\""xclip -i -selection clipboard <"\" -DDEFAULT_PASTE_FROM_CLIPBOARD_CMD=\""xclip -o -selection clipboard"\" -DDEFAULT_OPEN_FILE_UNDER_CURSOR_CMD=\""scopen"\" -DAUTOBACKUP -DHAVE_PTHREAD -DGNUPLOT -DODS -DXLSX -I/nix/store/8pznqjvlvmqll53cvmk9vmv6r45xp1wx-libxml2-2.12.7-dev/include/libxml2 -I/nix/store/83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/../83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/include   -c -o exec.o exec.c
clang -Wall -g -DNCURSES -D_XOPEN_SOURCE_EXTENDED -D_GNU_SOURCE -DSNAME=\"sc-im\" -DHELP_PATH=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/sc-im\" -DLIBDIR=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/doc/sc-im\" -DDFLT_PAGER=\"less\" -DDFLT_EDITOR=\"vim\" -DUSECOLORS -DHISTORY_FILE=\"sc-iminfo\" -DHISTORY_DIR=\".cache\" -DCONFIG_FILE=\"scimrc\" -DCONFIG_DIR=\".config/sc-im\" -DINS_HISTORY_FILE=\"sc-iminfo\" -DUNDO -DMAXROWS=65536 -DUSELOCALE -DMOUSE -DDEFAULT_COPY_TO_CLIPBOARD_CMD=\""xclip -i -selection clipboard <"\" -DDEFAULT_PASTE_FROM_CLIPBOARD_CMD=\""xclip -o -selection clipboard"\" -DDEFAULT_OPEN_FILE_UNDER_CURSOR_CMD=\""scopen"\" -DAUTOBACKUP -DHAVE_PTHREAD -DGNUPLOT -DODS -DXLSX -I/nix/store/8pznqjvlvmqll53cvmk9vmv6r45xp1wx-libxml2-2.12.7-dev/include/libxml2 -I/nix/store/83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/../83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/include   -c -o file.o file.c
clang -Wall -g -DNCURSES -D_XOPEN_SOURCE_EXTENDED -D_GNU_SOURCE -DSNAME=\"sc-im\" -DHELP_PATH=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/sc-im\" -DLIBDIR=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/doc/sc-im\" -DDFLT_PAGER=\"less\" -DDFLT_EDITOR=\"vim\" -DUSECOLORS -DHISTORY_FILE=\"sc-iminfo\" -DHISTORY_DIR=\".cache\" -DCONFIG_FILE=\"scimrc\" -DCONFIG_DIR=\".config/sc-im\" -DINS_HISTORY_FILE=\"sc-iminfo\" -DUNDO -DMAXROWS=65536 -DUSELOCALE -DMOUSE -DDEFAULT_COPY_TO_CLIPBOARD_CMD=\""xclip -i -selection clipboard <"\" -DDEFAULT_PASTE_FROM_CLIPBOARD_CMD=\""xclip -o -selection clipboard"\" -DDEFAULT_OPEN_FILE_UNDER_CURSOR_CMD=\""scopen"\" -DAUTOBACKUP -DHAVE_PTHREAD -DGNUPLOT -DODS -DXLSX -I/nix/store/8pznqjvlvmqll53cvmk9vmv6r45xp1wx-libxml2-2.12.7-dev/include/libxml2 -I/nix/store/83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/../83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/include   -c -o format.o format.c
clang -Wall -g -DNCURSES -D_XOPEN_SOURCE_EXTENDED -D_GNU_SOURCE -DSNAME=\"sc-im\" -DHELP_PATH=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/sc-im\" -DLIBDIR=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/doc/sc-im\" -DDFLT_PAGER=\"less\" -DDFLT_EDITOR=\"vim\" -DUSECOLORS -DHISTORY_FILE=\"sc-iminfo\" -DHISTORY_DIR=\".cache\" -DCONFIG_FILE=\"scimrc\" -DCONFIG_DIR=\".config/sc-im\" -DINS_HISTORY_FILE=\"sc-iminfo\" -DUNDO -DMAXROWS=65536 -DUSELOCALE -DMOUSE -DDEFAULT_COPY_TO_CLIPBOARD_CMD=\""xclip -i -selection clipboard <"\" -DDEFAULT_PASTE_FROM_CLIPBOARD_CMD=\""xclip -o -selection clipboard"\" -DDEFAULT_OPEN_FILE_UNDER_CURSOR_CMD=\""scopen"\" -DAUTOBACKUP -DHAVE_PTHREAD -DGNUPLOT -DODS -DXLSX -I/nix/store/8pznqjvlvmqll53cvmk9vmv6r45xp1wx-libxml2-2.12.7-dev/include/libxml2 -I/nix/store/83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/../83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/include   -c -o function.o function.c
clang -Wall -g -DNCURSES -D_XOPEN_SOURCE_EXTENDED -D_GNU_SOURCE -DSNAME=\"sc-im\" -DHELP_PATH=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/sc-im\" -DLIBDIR=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/doc/sc-im\" -DDFLT_PAGER=\"less\" -DDFLT_EDITOR=\"vim\" -DUSECOLORS -DHISTORY_FILE=\"sc-iminfo\" -DHISTORY_DIR=\".cache\" -DCONFIG_FILE=\"scimrc\" -DCONFIG_DIR=\".config/sc-im\" -DINS_HISTORY_FILE=\"sc-iminfo\" -DUNDO -DMAXROWS=65536 -DUSELOCALE -DMOUSE -DDEFAULT_COPY_TO_CLIPBOARD_CMD=\""xclip -i -selection clipboard <"\" -DDEFAULT_PASTE_FROM_CLIPBOARD_CMD=\""xclip -o -selection clipboard"\" -DDEFAULT_OPEN_FILE_UNDER_CURSOR_CMD=\""scopen"\" -DAUTOBACKUP -DHAVE_PTHREAD -DGNUPLOT -DODS -DXLSX -I/nix/store/8pznqjvlvmqll53cvmk9vmv6r45xp1wx-libxml2-2.12.7-dev/include/libxml2 -I/nix/store/83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/../83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/include   -c -o graph.o graph.c
clang -Wall -g -DNCURSES -D_XOPEN_SOURCE_EXTENDED -D_GNU_SOURCE -DSNAME=\"sc-im\" -DHELP_PATH=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/sc-im\" -DLIBDIR=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/doc/sc-im\" -DDFLT_PAGER=\"less\" -DDFLT_EDITOR=\"vim\" -DUSECOLORS -DHISTORY_FILE=\"sc-iminfo\" -DHISTORY_DIR=\".cache\" -DCONFIG_FILE=\"scimrc\" -DCONFIG_DIR=\".config/sc-im\" -DINS_HISTORY_FILE=\"sc-iminfo\" -DUNDO -DMAXROWS=65536 -DUSELOCALE -DMOUSE -DDEFAULT_COPY_TO_CLIPBOARD_CMD=\""xclip -i -selection clipboard <"\" -DDEFAULT_PASTE_FROM_CLIPBOARD_CMD=\""xclip -o -selection clipboard"\" -DDEFAULT_OPEN_FILE_UNDER_CURSOR_CMD=\""scopen"\" -DAUTOBACKUP -DHAVE_PTHREAD -DGNUPLOT -DODS -DXLSX -I/nix/store/8pznqjvlvmqll53cvmk9vmv6r45xp1wx-libxml2-2.12.7-dev/include/libxml2 -I/nix/store/83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/../83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/include   -c -o help.o help.c
clang -Wall -g -DNCURSES -D_XOPEN_SOURCE_EXTENDED -D_GNU_SOURCE -DSNAME=\"sc-im\" -DHELP_PATH=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/sc-im\" -DLIBDIR=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/doc/sc-im\" -DDFLT_PAGER=\"less\" -DDFLT_EDITOR=\"vim\" -DUSECOLORS -DHISTORY_FILE=\"sc-iminfo\" -DHISTORY_DIR=\".cache\" -DCONFIG_FILE=\"scimrc\" -DCONFIG_DIR=\".config/sc-im\" -DINS_HISTORY_FILE=\"sc-iminfo\" -DUNDO -DMAXROWS=65536 -DUSELOCALE -DMOUSE -DDEFAULT_COPY_TO_CLIPBOARD_CMD=\""xclip -i -selection clipboard <"\" -DDEFAULT_PASTE_FROM_CLIPBOARD_CMD=\""xclip -o -selection clipboard"\" -DDEFAULT_OPEN_FILE_UNDER_CURSOR_CMD=\""scopen"\" -DAUTOBACKUP -DHAVE_PTHREAD -DGNUPLOT -DODS -DXLSX -I/nix/store/8pznqjvlvmqll53cvmk9vmv6r45xp1wx-libxml2-2.12.7-dev/include/libxml2 -I/nix/store/83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/../83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/include   -c -o history.o history.c
clang -Wall -g -DNCURSES -D_XOPEN_SOURCE_EXTENDED -D_GNU_SOURCE -DSNAME=\"sc-im\" -DHELP_PATH=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/sc-im\" -DLIBDIR=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/doc/sc-im\" -DDFLT_PAGER=\"less\" -DDFLT_EDITOR=\"vim\" -DUSECOLORS -DHISTORY_FILE=\"sc-iminfo\" -DHISTORY_DIR=\".cache\" -DCONFIG_FILE=\"scimrc\" -DCONFIG_DIR=\".config/sc-im\" -DINS_HISTORY_FILE=\"sc-iminfo\" -DUNDO -DMAXROWS=65536 -DUSELOCALE -DMOUSE -DDEFAULT_COPY_TO_CLIPBOARD_CMD=\""xclip -i -selection clipboard <"\" -DDEFAULT_PASTE_FROM_CLIPBOARD_CMD=\""xclip -o -selection clipboard"\" -DDEFAULT_OPEN_FILE_UNDER_CURSOR_CMD=\""scopen"\" -DAUTOBACKUP -DHAVE_PTHREAD -DGNUPLOT -DODS -DXLSX -I/nix/store/8pznqjvlvmqll53cvmk9vmv6r45xp1wx-libxml2-2.12.7-dev/include/libxml2 -I/nix/store/83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/../83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/include   -c -o input.o input.c
clang -Wall -g -DNCURSES -D_XOPEN_SOURCE_EXTENDED -D_GNU_SOURCE -DSNAME=\"sc-im\" -DHELP_PATH=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/sc-im\" -DLIBDIR=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/doc/sc-im\" -DDFLT_PAGER=\"less\" -DDFLT_EDITOR=\"vim\" -DUSECOLORS -DHISTORY_FILE=\"sc-iminfo\" -DHISTORY_DIR=\".cache\" -DCONFIG_FILE=\"scimrc\" -DCONFIG_DIR=\".config/sc-im\" -DINS_HISTORY_FILE=\"sc-iminfo\" -DUNDO -DMAXROWS=65536 -DUSELOCALE -DMOUSE -DDEFAULT_COPY_TO_CLIPBOARD_CMD=\""xclip -i -selection clipboard <"\" -DDEFAULT_PASTE_FROM_CLIPBOARD_CMD=\""xclip -o -selection clipboard"\" -DDEFAULT_OPEN_FILE_UNDER_CURSOR_CMD=\""scopen"\" -DAUTOBACKUP -DHAVE_PTHREAD -DGNUPLOT -DODS -DXLSX -I/nix/store/8pznqjvlvmqll53cvmk9vmv6r45xp1wx-libxml2-2.12.7-dev/include/libxml2 -I/nix/store/83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/../83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/include   -c -o interp.o interp.c
interp.c:787:17: warning: passing arguments to a function without a prototype is deprecated in all versions of C and is not supported in C2x [-Wdeprecated-non-prototype]
    res = (*fn) (arg);
                ^
interp.c:804:17: warning: passing arguments to a function without a prototype is deprecated in all versions of C and is not supported in C2x [-Wdeprecated-non-prototype]
    res = (*fn) (arg1, arg2);
                ^
2 warnings generated.
clang -Wall -g -DNCURSES -D_XOPEN_SOURCE_EXTENDED -D_GNU_SOURCE -DSNAME=\"sc-im\" -DHELP_PATH=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/sc-im\" -DLIBDIR=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/doc/sc-im\" -DDFLT_PAGER=\"less\" -DDFLT_EDITOR=\"vim\" -DUSECOLORS -DHISTORY_FILE=\"sc-iminfo\" -DHISTORY_DIR=\".cache\" -DCONFIG_FILE=\"scimrc\" -DCONFIG_DIR=\".config/sc-im\" -DINS_HISTORY_FILE=\"sc-iminfo\" -DUNDO -DMAXROWS=65536 -DUSELOCALE -DMOUSE -DDEFAULT_COPY_TO_CLIPBOARD_CMD=\""xclip -i -selection clipboard <"\" -DDEFAULT_PASTE_FROM_CLIPBOARD_CMD=\""xclip -o -selection clipboard"\" -DDEFAULT_OPEN_FILE_UNDER_CURSOR_CMD=\""scopen"\" -DAUTOBACKUP -DHAVE_PTHREAD -DGNUPLOT -DODS -DXLSX -I/nix/store/8pznqjvlvmqll53cvmk9vmv6r45xp1wx-libxml2-2.12.7-dev/include/libxml2 -I/nix/store/83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/../83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/include   -c -o lex.o lex.c
lex.c:270:25: warning: misleading indentation; statement is not part of the previous 'while' [-Wmisleading-indentation]
                        if (isalpha(*p) || *p == '_') {
                        ^
lex.c:268:21: note: previous statement is here
                    while (isdigit(*++p))
                    ^
1 warning generated.
clang -Wall -g -DNCURSES -D_XOPEN_SOURCE_EXTENDED -D_GNU_SOURCE -DSNAME=\"sc-im\" -DHELP_PATH=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/sc-im\" -DLIBDIR=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/doc/sc-im\" -DDFLT_PAGER=\"less\" -DDFLT_EDITOR=\"vim\" -DUSECOLORS -DHISTORY_FILE=\"sc-iminfo\" -DHISTORY_DIR=\".cache\" -DCONFIG_FILE=\"scimrc\" -DCONFIG_DIR=\".config/sc-im\" -DINS_HISTORY_FILE=\"sc-iminfo\" -DUNDO -DMAXROWS=65536 -DUSELOCALE -DMOUSE -DDEFAULT_COPY_TO_CLIPBOARD_CMD=\""xclip -i -selection clipboard <"\" -DDEFAULT_PASTE_FROM_CLIPBOARD_CMD=\""xclip -o -selection clipboard"\" -DDEFAULT_OPEN_FILE_UNDER_CURSOR_CMD=\""scopen"\" -DAUTOBACKUP -DHAVE_PTHREAD -DGNUPLOT -DODS -DXLSX -I/nix/store/8pznqjvlvmqll53cvmk9vmv6r45xp1wx-libxml2-2.12.7-dev/include/libxml2 -I/nix/store/83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/../83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/include   -c -o lua.o lua.c
clang -Wall -g -DNCURSES -D_XOPEN_SOURCE_EXTENDED -D_GNU_SOURCE -DSNAME=\"sc-im\" -DHELP_PATH=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/sc-im\" -DLIBDIR=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/doc/sc-im\" -DDFLT_PAGER=\"less\" -DDFLT_EDITOR=\"vim\" -DUSECOLORS -DHISTORY_FILE=\"sc-iminfo\" -DHISTORY_DIR=\".cache\" -DCONFIG_FILE=\"scimrc\" -DCONFIG_DIR=\".config/sc-im\" -DINS_HISTORY_FILE=\"sc-iminfo\" -DUNDO -DMAXROWS=65536 -DUSELOCALE -DMOUSE -DDEFAULT_COPY_TO_CLIPBOARD_CMD=\""xclip -i -selection clipboard <"\" -DDEFAULT_PASTE_FROM_CLIPBOARD_CMD=\""xclip -o -selection clipboard"\" -DDEFAULT_OPEN_FILE_UNDER_CURSOR_CMD=\""scopen"\" -DAUTOBACKUP -DHAVE_PTHREAD -DGNUPLOT -DODS -DXLSX -I/nix/store/8pznqjvlvmqll53cvmk9vmv6r45xp1wx-libxml2-2.12.7-dev/include/libxml2 -I/nix/store/83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/../83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/include   -c -o main.o main.c
clang -Wall -g -DNCURSES -D_XOPEN_SOURCE_EXTENDED -D_GNU_SOURCE -DSNAME=\"sc-im\" -DHELP_PATH=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/sc-im\" -DLIBDIR=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/doc/sc-im\" -DDFLT_PAGER=\"less\" -DDFLT_EDITOR=\"vim\" -DUSECOLORS -DHISTORY_FILE=\"sc-iminfo\" -DHISTORY_DIR=\".cache\" -DCONFIG_FILE=\"scimrc\" -DCONFIG_DIR=\".config/sc-im\" -DINS_HISTORY_FILE=\"sc-iminfo\" -DUNDO -DMAXROWS=65536 -DUSELOCALE -DMOUSE -DDEFAULT_COPY_TO_CLIPBOARD_CMD=\""xclip -i -selection clipboard <"\" -DDEFAULT_PASTE_FROM_CLIPBOARD_CMD=\""xclip -o -selection clipboard"\" -DDEFAULT_OPEN_FILE_UNDER_CURSOR_CMD=\""scopen"\" -DAUTOBACKUP -DHAVE_PTHREAD -DGNUPLOT -DODS -DXLSX -I/nix/store/8pznqjvlvmqll53cvmk9vmv6r45xp1wx-libxml2-2.12.7-dev/include/libxml2 -I/nix/store/83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/../83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/include   -c -o maps.o maps.c
clang -Wall -g -DNCURSES -D_XOPEN_SOURCE_EXTENDED -D_GNU_SOURCE -DSNAME=\"sc-im\" -DHELP_PATH=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/sc-im\" -DLIBDIR=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/doc/sc-im\" -DDFLT_PAGER=\"less\" -DDFLT_EDITOR=\"vim\" -DUSECOLORS -DHISTORY_FILE=\"sc-iminfo\" -DHISTORY_DIR=\".cache\" -DCONFIG_FILE=\"scimrc\" -DCONFIG_DIR=\".config/sc-im\" -DINS_HISTORY_FILE=\"sc-iminfo\" -DUNDO -DMAXROWS=65536 -DUSELOCALE -DMOUSE -DDEFAULT_COPY_TO_CLIPBOARD_CMD=\""xclip -i -selection clipboard <"\" -DDEFAULT_PASTE_FROM_CLIPBOARD_CMD=\""xclip -o -selection clipboard"\" -DDEFAULT_OPEN_FILE_UNDER_CURSOR_CMD=\""scopen"\" -DAUTOBACKUP -DHAVE_PTHREAD -DGNUPLOT -DODS -DXLSX -I/nix/store/8pznqjvlvmqll53cvmk9vmv6r45xp1wx-libxml2-2.12.7-dev/include/libxml2 -I/nix/store/83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/../83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/include   -c -o marks.o marks.c
clang -Wall -g -DNCURSES -D_XOPEN_SOURCE_EXTENDED -D_GNU_SOURCE -DSNAME=\"sc-im\" -DHELP_PATH=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/sc-im\" -DLIBDIR=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/doc/sc-im\" -DDFLT_PAGER=\"less\" -DDFLT_EDITOR=\"vim\" -DUSECOLORS -DHISTORY_FILE=\"sc-iminfo\" -DHISTORY_DIR=\".cache\" -DCONFIG_FILE=\"scimrc\" -DCONFIG_DIR=\".config/sc-im\" -DINS_HISTORY_FILE=\"sc-iminfo\" -DUNDO -DMAXROWS=65536 -DUSELOCALE -DMOUSE -DDEFAULT_COPY_TO_CLIPBOARD_CMD=\""xclip -i -selection clipboard <"\" -DDEFAULT_PASTE_FROM_CLIPBOARD_CMD=\""xclip -o -selection clipboard"\" -DDEFAULT_OPEN_FILE_UNDER_CURSOR_CMD=\""scopen"\" -DAUTOBACKUP -DHAVE_PTHREAD -DGNUPLOT -DODS -DXLSX -I/nix/store/8pznqjvlvmqll53cvmk9vmv6r45xp1wx-libxml2-2.12.7-dev/include/libxml2 -I/nix/store/83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/../83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/include   -c -o pipe.o pipe.c
clang -Wall -g -DNCURSES -D_XOPEN_SOURCE_EXTENDED -D_GNU_SOURCE -DSNAME=\"sc-im\" -DHELP_PATH=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/sc-im\" -DLIBDIR=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/doc/sc-im\" -DDFLT_PAGER=\"less\" -DDFLT_EDITOR=\"vim\" -DUSECOLORS -DHISTORY_FILE=\"sc-iminfo\" -DHISTORY_DIR=\".cache\" -DCONFIG_FILE=\"scimrc\" -DCONFIG_DIR=\".config/sc-im\" -DINS_HISTORY_FILE=\"sc-iminfo\" -DUNDO -DMAXROWS=65536 -DUSELOCALE -DMOUSE -DDEFAULT_COPY_TO_CLIPBOARD_CMD=\""xclip -i -selection clipboard <"\" -DDEFAULT_PASTE_FROM_CLIPBOARD_CMD=\""xclip -o -selection clipboard"\" -DDEFAULT_OPEN_FILE_UNDER_CURSOR_CMD=\""scopen"\" -DAUTOBACKUP -DHAVE_PTHREAD -DGNUPLOT -DODS -DXLSX -I/nix/store/8pznqjvlvmqll53cvmk9vmv6r45xp1wx-libxml2-2.12.7-dev/include/libxml2 -I/nix/store/83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/../83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/include   -c -o range.o range.c
clang -Wall -g -DNCURSES -D_XOPEN_SOURCE_EXTENDED -D_GNU_SOURCE -DSNAME=\"sc-im\" -DHELP_PATH=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/sc-im\" -DLIBDIR=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/doc/sc-im\" -DDFLT_PAGER=\"less\" -DDFLT_EDITOR=\"vim\" -DUSECOLORS -DHISTORY_FILE=\"sc-iminfo\" -DHISTORY_DIR=\".cache\" -DCONFIG_FILE=\"scimrc\" -DCONFIG_DIR=\".config/sc-im\" -DINS_HISTORY_FILE=\"sc-iminfo\" -DUNDO -DMAXROWS=65536 -DUSELOCALE -DMOUSE -DDEFAULT_COPY_TO_CLIPBOARD_CMD=\""xclip -i -selection clipboard <"\" -DDEFAULT_PASTE_FROM_CLIPBOARD_CMD=\""xclip -o -selection clipboard"\" -DDEFAULT_OPEN_FILE_UNDER_CURSOR_CMD=\""scopen"\" -DAUTOBACKUP -DHAVE_PTHREAD -DGNUPLOT -DODS -DXLSX -I/nix/store/8pznqjvlvmqll53cvmk9vmv6r45xp1wx-libxml2-2.12.7-dev/include/libxml2 -I/nix/store/83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/../83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/include   -c -o sheet.o sheet.c
clang -Wall -g -DNCURSES -D_XOPEN_SOURCE_EXTENDED -D_GNU_SOURCE -DSNAME=\"sc-im\" -DHELP_PATH=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/sc-im\" -DLIBDIR=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/doc/sc-im\" -DDFLT_PAGER=\"less\" -DDFLT_EDITOR=\"vim\" -DUSECOLORS -DHISTORY_FILE=\"sc-iminfo\" -DHISTORY_DIR=\".cache\" -DCONFIG_FILE=\"scimrc\" -DCONFIG_DIR=\".config/sc-im\" -DINS_HISTORY_FILE=\"sc-iminfo\" -DUNDO -DMAXROWS=65536 -DUSELOCALE -DMOUSE -DDEFAULT_COPY_TO_CLIPBOARD_CMD=\""xclip -i -selection clipboard <"\" -DDEFAULT_PASTE_FROM_CLIPBOARD_CMD=\""xclip -o -selection clipboard"\" -DDEFAULT_OPEN_FILE_UNDER_CURSOR_CMD=\""scopen"\" -DAUTOBACKUP -DHAVE_PTHREAD -DGNUPLOT -DODS -DXLSX -I/nix/store/8pznqjvlvmqll53cvmk9vmv6r45xp1wx-libxml2-2.12.7-dev/include/libxml2 -I/nix/store/83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/../83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/include   -c -o trigger.o trigger.c
clang -Wall -g -DNCURSES -D_XOPEN_SOURCE_EXTENDED -D_GNU_SOURCE -DSNAME=\"sc-im\" -DHELP_PATH=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/sc-im\" -DLIBDIR=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/doc/sc-im\" -DDFLT_PAGER=\"less\" -DDFLT_EDITOR=\"vim\" -DUSECOLORS -DHISTORY_FILE=\"sc-iminfo\" -DHISTORY_DIR=\".cache\" -DCONFIG_FILE=\"scimrc\" -DCONFIG_DIR=\".config/sc-im\" -DINS_HISTORY_FILE=\"sc-iminfo\" -DUNDO -DMAXROWS=65536 -DUSELOCALE -DMOUSE -DDEFAULT_COPY_TO_CLIPBOARD_CMD=\""xclip -i -selection clipboard <"\" -DDEFAULT_PASTE_FROM_CLIPBOARD_CMD=\""xclip -o -selection clipboard"\" -DDEFAULT_OPEN_FILE_UNDER_CURSOR_CMD=\""scopen"\" -DAUTOBACKUP -DHAVE_PTHREAD -DGNUPLOT -DODS -DXLSX -I/nix/store/8pznqjvlvmqll53cvmk9vmv6r45xp1wx-libxml2-2.12.7-dev/include/libxml2 -I/nix/store/83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/../83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/include   -c -o tui.o tui.c
clang -Wall -g -DNCURSES -D_XOPEN_SOURCE_EXTENDED -D_GNU_SOURCE -DSNAME=\"sc-im\" -DHELP_PATH=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/sc-im\" -DLIBDIR=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/doc/sc-im\" -DDFLT_PAGER=\"less\" -DDFLT_EDITOR=\"vim\" -DUSECOLORS -DHISTORY_FILE=\"sc-iminfo\" -DHISTORY_DIR=\".cache\" -DCONFIG_FILE=\"scimrc\" -DCONFIG_DIR=\".config/sc-im\" -DINS_HISTORY_FILE=\"sc-iminfo\" -DUNDO -DMAXROWS=65536 -DUSELOCALE -DMOUSE -DDEFAULT_COPY_TO_CLIPBOARD_CMD=\""xclip -i -selection clipboard <"\" -DDEFAULT_PASTE_FROM_CLIPBOARD_CMD=\""xclip -o -selection clipboard"\" -DDEFAULT_OPEN_FILE_UNDER_CURSOR_CMD=\""scopen"\" -DAUTOBACKUP -DHAVE_PTHREAD -DGNUPLOT -DODS -DXLSX -I/nix/store/8pznqjvlvmqll53cvmk9vmv6r45xp1wx-libxml2-2.12.7-dev/include/libxml2 -I/nix/store/83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/../83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/include   -c -o undo.o undo.c
clang -Wall -g -DNCURSES -D_XOPEN_SOURCE_EXTENDED -D_GNU_SOURCE -DSNAME=\"sc-im\" -DHELP_PATH=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/sc-im\" -DLIBDIR=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/doc/sc-im\" -DDFLT_PAGER=\"less\" -DDFLT_EDITOR=\"vim\" -DUSECOLORS -DHISTORY_FILE=\"sc-iminfo\" -DHISTORY_DIR=\".cache\" -DCONFIG_FILE=\"scimrc\" -DCONFIG_DIR=\".config/sc-im\" -DINS_HISTORY_FILE=\"sc-iminfo\" -DUNDO -DMAXROWS=65536 -DUSELOCALE -DMOUSE -DDEFAULT_COPY_TO_CLIPBOARD_CMD=\""xclip -i -selection clipboard <"\" -DDEFAULT_PASTE_FROM_CLIPBOARD_CMD=\""xclip -o -selection clipboard"\" -DDEFAULT_OPEN_FILE_UNDER_CURSOR_CMD=\""scopen"\" -DAUTOBACKUP -DHAVE_PTHREAD -DGNUPLOT -DODS -DXLSX -I/nix/store/8pznqjvlvmqll53cvmk9vmv6r45xp1wx-libxml2-2.12.7-dev/include/libxml2 -I/nix/store/83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/../83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/include   -c -o vmtbl.o vmtbl.c
clang -Wall -g -DNCURSES -D_XOPEN_SOURCE_EXTENDED -D_GNU_SOURCE -DSNAME=\"sc-im\" -DHELP_PATH=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/sc-im\" -DLIBDIR=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/doc/sc-im\" -DDFLT_PAGER=\"less\" -DDFLT_EDITOR=\"vim\" -DUSECOLORS -DHISTORY_FILE=\"sc-iminfo\" -DHISTORY_DIR=\".cache\" -DCONFIG_FILE=\"scimrc\" -DCONFIG_DIR=\".config/sc-im\" -DINS_HISTORY_FILE=\"sc-iminfo\" -DUNDO -DMAXROWS=65536 -DUSELOCALE -DMOUSE -DDEFAULT_COPY_TO_CLIPBOARD_CMD=\""xclip -i -selection clipboard <"\" -DDEFAULT_PASTE_FROM_CLIPBOARD_CMD=\""xclip -o -selection clipboard"\" -DDEFAULT_OPEN_FILE_UNDER_CURSOR_CMD=\""scopen"\" -DAUTOBACKUP -DHAVE_PTHREAD -DGNUPLOT -DODS -DXLSX -I/nix/store/8pznqjvlvmqll53cvmk9vmv6r45xp1wx-libxml2-2.12.7-dev/include/libxml2 -I/nix/store/83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/../83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/include   -c -o xmalloc.o xmalloc.c
xmalloc.c:57:13: warning: a function declaration without a prototype is deprecated in all versions of C and is treated as a zero-parameter prototype in C2x, conflicting with a previous declaration [-Wdeprecated-non-prototype]
extern void free();
            ^
1 warning generated.
clang -Wall -g -DNCURSES -D_XOPEN_SOURCE_EXTENDED -D_GNU_SOURCE -DSNAME=\"sc-im\" -DHELP_PATH=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/sc-im\" -DLIBDIR=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/doc/sc-im\" -DDFLT_PAGER=\"less\" -DDFLT_EDITOR=\"vim\" -DUSECOLORS -DHISTORY_FILE=\"sc-iminfo\" -DHISTORY_DIR=\".cache\" -DCONFIG_FILE=\"scimrc\" -DCONFIG_DIR=\".config/sc-im\" -DINS_HISTORY_FILE=\"sc-iminfo\" -DUNDO -DMAXROWS=65536 -DUSELOCALE -DMOUSE -DDEFAULT_COPY_TO_CLIPBOARD_CMD=\""xclip -i -selection clipboard <"\" -DDEFAULT_PASTE_FROM_CLIPBOARD_CMD=\""xclip -o -selection clipboard"\" -DDEFAULT_OPEN_FILE_UNDER_CURSOR_CMD=\""scopen"\" -DAUTOBACKUP -DHAVE_PTHREAD -DGNUPLOT -DODS -DXLSX -I/nix/store/8pznqjvlvmqll53cvmk9vmv6r45xp1wx-libxml2-2.12.7-dev/include/libxml2 -I/nix/store/83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/../83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/include   -c -o yank.o yank.c
clang -Wall -g -DNCURSES -D_XOPEN_SOURCE_EXTENDED -D_GNU_SOURCE -DSNAME=\"sc-im\" -DHELP_PATH=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/sc-im\" -DLIBDIR=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/doc/sc-im\" -DDFLT_PAGER=\"less\" -DDFLT_EDITOR=\"vim\" -DUSECOLORS -DHISTORY_FILE=\"sc-iminfo\" -DHISTORY_DIR=\".cache\" -DCONFIG_FILE=\"scimrc\" -DCONFIG_DIR=\".config/sc-im\" -DINS_HISTORY_FILE=\"sc-iminfo\" -DUNDO -DMAXROWS=65536 -DUSELOCALE -DMOUSE -DDEFAULT_COPY_TO_CLIPBOARD_CMD=\""xclip -i -selection clipboard <"\" -DDEFAULT_PASTE_FROM_CLIPBOARD_CMD=\""xclip -o -selection clipboard"\" -DDEFAULT_OPEN_FILE_UNDER_CURSOR_CMD=\""scopen"\" -DAUTOBACKUP -DHAVE_PTHREAD -DGNUPLOT -DODS -DXLSX -I/nix/store/8pznqjvlvmqll53cvmk9vmv6r45xp1wx-libxml2-2.12.7-dev/include/libxml2 -I/nix/store/83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/../83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/include   -c -o actions/filter.o actions/filter.c
clang -Wall -g -DNCURSES -D_XOPEN_SOURCE_EXTENDED -D_GNU_SOURCE -DSNAME=\"sc-im\" -DHELP_PATH=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/sc-im\" -DLIBDIR=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/doc/sc-im\" -DDFLT_PAGER=\"less\" -DDFLT_EDITOR=\"vim\" -DUSECOLORS -DHISTORY_FILE=\"sc-iminfo\" -DHISTORY_DIR=\".cache\" -DCONFIG_FILE=\"scimrc\" -DCONFIG_DIR=\".config/sc-im\" -DINS_HISTORY_FILE=\"sc-iminfo\" -DUNDO -DMAXROWS=65536 -DUSELOCALE -DMOUSE -DDEFAULT_COPY_TO_CLIPBOARD_CMD=\""xclip -i -selection clipboard <"\" -DDEFAULT_PASTE_FROM_CLIPBOARD_CMD=\""xclip -o -selection clipboard"\" -DDEFAULT_OPEN_FILE_UNDER_CURSOR_CMD=\""scopen"\" -DAUTOBACKUP -DHAVE_PTHREAD -DGNUPLOT -DODS -DXLSX -I/nix/store/8pznqjvlvmqll53cvmk9vmv6r45xp1wx-libxml2-2.12.7-dev/include/libxml2 -I/nix/store/83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/../83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/include   -c -o actions/freeze.o actions/freeze.c
clang -Wall -g -DNCURSES -D_XOPEN_SOURCE_EXTENDED -D_GNU_SOURCE -DSNAME=\"sc-im\" -DHELP_PATH=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/sc-im\" -DLIBDIR=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/doc/sc-im\" -DDFLT_PAGER=\"less\" -DDFLT_EDITOR=\"vim\" -DUSECOLORS -DHISTORY_FILE=\"sc-iminfo\" -DHISTORY_DIR=\".cache\" -DCONFIG_FILE=\"scimrc\" -DCONFIG_DIR=\".config/sc-im\" -DINS_HISTORY_FILE=\"sc-iminfo\" -DUNDO -DMAXROWS=65536 -DUSELOCALE -DMOUSE -DDEFAULT_COPY_TO_CLIPBOARD_CMD=\""xclip -i -selection clipboard <"\" -DDEFAULT_PASTE_FROM_CLIPBOARD_CMD=\""xclip -o -selection clipboard"\" -DDEFAULT_OPEN_FILE_UNDER_CURSOR_CMD=\""scopen"\" -DAUTOBACKUP -DHAVE_PTHREAD -DGNUPLOT -DODS -DXLSX -I/nix/store/8pznqjvlvmqll53cvmk9vmv6r45xp1wx-libxml2-2.12.7-dev/include/libxml2 -I/nix/store/83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/../83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/include   -c -o actions/hide_show.o actions/hide_show.c
actions/hide_show.c:250:12: warning: variable 'count' set but not used [-Wunused-but-set-variable]
    int c, count = 0;
           ^
1 warning generated.
clang -Wall -g -DNCURSES -D_XOPEN_SOURCE_EXTENDED -D_GNU_SOURCE -DSNAME=\"sc-im\" -DHELP_PATH=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/sc-im\" -DLIBDIR=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/doc/sc-im\" -DDFLT_PAGER=\"less\" -DDFLT_EDITOR=\"vim\" -DUSECOLORS -DHISTORY_FILE=\"sc-iminfo\" -DHISTORY_DIR=\".cache\" -DCONFIG_FILE=\"scimrc\" -DCONFIG_DIR=\".config/sc-im\" -DINS_HISTORY_FILE=\"sc-iminfo\" -DUNDO -DMAXROWS=65536 -DUSELOCALE -DMOUSE -DDEFAULT_COPY_TO_CLIPBOARD_CMD=\""xclip -i -selection clipboard <"\" -DDEFAULT_PASTE_FROM_CLIPBOARD_CMD=\""xclip -o -selection clipboard"\" -DDEFAULT_OPEN_FILE_UNDER_CURSOR_CMD=\""scopen"\" -DAUTOBACKUP -DHAVE_PTHREAD -DGNUPLOT -DODS -DXLSX -I/nix/store/8pznqjvlvmqll53cvmk9vmv6r45xp1wx-libxml2-2.12.7-dev/include/libxml2 -I/nix/store/83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/../83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/include   -c -o actions/plot.o actions/plot.c
clang -Wall -g -DNCURSES -D_XOPEN_SOURCE_EXTENDED -D_GNU_SOURCE -DSNAME=\"sc-im\" -DHELP_PATH=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/sc-im\" -DLIBDIR=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/doc/sc-im\" -DDFLT_PAGER=\"less\" -DDFLT_EDITOR=\"vim\" -DUSECOLORS -DHISTORY_FILE=\"sc-iminfo\" -DHISTORY_DIR=\".cache\" -DCONFIG_FILE=\"scimrc\" -DCONFIG_DIR=\".config/sc-im\" -DINS_HISTORY_FILE=\"sc-iminfo\" -DUNDO -DMAXROWS=65536 -DUSELOCALE -DMOUSE -DDEFAULT_COPY_TO_CLIPBOARD_CMD=\""xclip -i -selection clipboard <"\" -DDEFAULT_PASTE_FROM_CLIPBOARD_CMD=\""xclip -o -selection clipboard"\" -DDEFAULT_OPEN_FILE_UNDER_CURSOR_CMD=\""scopen"\" -DAUTOBACKUP -DHAVE_PTHREAD -DGNUPLOT -DODS -DXLSX -I/nix/store/8pznqjvlvmqll53cvmk9vmv6r45xp1wx-libxml2-2.12.7-dev/include/libxml2 -I/nix/store/83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/../83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/include   -c -o actions/shift.o actions/shift.c
clang -Wall -g -DNCURSES -D_XOPEN_SOURCE_EXTENDED -D_GNU_SOURCE -DSNAME=\"sc-im\" -DHELP_PATH=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/sc-im\" -DLIBDIR=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/doc/sc-im\" -DDFLT_PAGER=\"less\" -DDFLT_EDITOR=\"vim\" -DUSECOLORS -DHISTORY_FILE=\"sc-iminfo\" -DHISTORY_DIR=\".cache\" -DCONFIG_FILE=\"scimrc\" -DCONFIG_DIR=\".config/sc-im\" -DINS_HISTORY_FILE=\"sc-iminfo\" -DUNDO -DMAXROWS=65536 -DUSELOCALE -DMOUSE -DDEFAULT_COPY_TO_CLIPBOARD_CMD=\""xclip -i -selection clipboard <"\" -DDEFAULT_PASTE_FROM_CLIPBOARD_CMD=\""xclip -o -selection clipboard"\" -DDEFAULT_OPEN_FILE_UNDER_CURSOR_CMD=\""scopen"\" -DAUTOBACKUP -DHAVE_PTHREAD -DGNUPLOT -DODS -DXLSX -I/nix/store/8pznqjvlvmqll53cvmk9vmv6r45xp1wx-libxml2-2.12.7-dev/include/libxml2 -I/nix/store/83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/../83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/include   -c -o actions/sort.o actions/sort.c
clang -Wall -g -DNCURSES -D_XOPEN_SOURCE_EXTENDED -D_GNU_SOURCE -DSNAME=\"sc-im\" -DHELP_PATH=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/sc-im\" -DLIBDIR=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/doc/sc-im\" -DDFLT_PAGER=\"less\" -DDFLT_EDITOR=\"vim\" -DUSECOLORS -DHISTORY_FILE=\"sc-iminfo\" -DHISTORY_DIR=\".cache\" -DCONFIG_FILE=\"scimrc\" -DCONFIG_DIR=\".config/sc-im\" -DINS_HISTORY_FILE=\"sc-iminfo\" -DUNDO -DMAXROWS=65536 -DUSELOCALE -DMOUSE -DDEFAULT_COPY_TO_CLIPBOARD_CMD=\""xclip -i -selection clipboard <"\" -DDEFAULT_PASTE_FROM_CLIPBOARD_CMD=\""xclip -o -selection clipboard"\" -DDEFAULT_OPEN_FILE_UNDER_CURSOR_CMD=\""scopen"\" -DAUTOBACKUP -DHAVE_PTHREAD -DGNUPLOT -DODS -DXLSX -I/nix/store/8pznqjvlvmqll53cvmk9vmv6r45xp1wx-libxml2-2.12.7-dev/include/libxml2 -I/nix/store/83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/../83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/include   -c -o actions/subtotal.o actions/subtotal.c
clang -Wall -g -DNCURSES -D_XOPEN_SOURCE_EXTENDED -D_GNU_SOURCE -DSNAME=\"sc-im\" -DHELP_PATH=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/sc-im\" -DLIBDIR=\"/nix/store/xc58gdc52skvdqbw8q16zh0d89ic9f9d-sc-im-0.8.3/share/doc/sc-im\" -DDFLT_PAGER=\"less\" -DDFLT_EDITOR=\"vim\" -DUSECOLORS -DHISTORY_FILE=\"sc-iminfo\" -DHISTORY_DIR=\".cache\" -DCONFIG_FILE=\"scimrc\" -DCONFIG_DIR=\".config/sc-im\" -DINS_HISTORY_FILE=\"sc-iminfo\" -DUNDO -DMAXROWS=65536 -DUSELOCALE -DMOUSE -DDEFAULT_COPY_TO_CLIPBOARD_CMD=\""xclip -i -selection clipboard <"\" -DDEFAULT_PASTE_FROM_CLIPBOARD_CMD=\""xclip -o -selection clipboard"\" -DDEFAULT_OPEN_FILE_UNDER_CURSOR_CMD=\""scopen"\" -DAUTOBACKUP -DHAVE_PTHREAD -DGNUPLOT -DODS -DXLSX -I/nix/store/8pznqjvlvmqll53cvmk9vmv6r45xp1wx-libxml2-2.12.7-dev/include/libxml2 -I/nix/store/83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/../83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/include   -c -o formats/ods.o formats/ods.c
formats/ods.c:85:9: warning: 'zip_error_to_str' is deprecated: use 'zip_error_strerror' instead [-Wdeprecated-declarations]
        zip_error_to_str(buf, sizeof(buf), err, errno);
        ^
/nix/store/83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/include/zip.h:389:1: note: 'zip_error_to_str' has been explicitly marked deprecated here
ZIP_DEPRECATED("use 'zip_error_strerror' instead") ZIP_EXTERN int zip_error_to_str(char *_Nonnull, zip_uint64_t, int, int);
^
/nix/store/83b6bnkzq132slg9y4k9v3pm2dj689p1-libzip-1.10.1-dev/include/zip.h:63:42: note: expanded from macro 'ZIP_DEPRECATED'
#define ZIP_DEPRECATED(x) __attribute__((deprecated(x)))
                                         ^
formats/ods.c:97:28: error: call to undeclared library function 'malloc' with type 'void *(unsigned long)'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
        content = (char *) malloc(sb_content.size);
                           ^
formats/ods.c:97:28: note: include the header <stdlib.h> or explicitly provide a declaration for 'malloc'
formats/ods.c:101:13: error: call to undeclared library function 'free' with type 'void (void *)'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
            free(content);
            ^
formats/ods.c:101:13: note: include the header <stdlib.h> or explicitly provide a declaration for 'free'
formats/ods.c:211:35: error: call to undeclared function 'atof'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
                       double l = atof((char *) value);
                                  ^
1 warning and 3 errors generated.
make: *** [<builtin>: formats/ods.o] Error 1
error: builder for '/nix/store/pinl7arfp0w0d5kq48xdsvsz1la8d1v7-sc-im-0.8.3.drv' failed with exit code 2;
       last 10 log lines:
       > formats/ods.c:97:28: note: include the header <stdlib.h> or explicitly provide a declaration for 'malloc'
       > formats/ods.c:101:13: error: call to undeclared library function 'free' with type 'void (void *)'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
       >             free(content);
       >             ^
       > formats/ods.c:101:13: note: include the header <stdlib.h> or explicitly provide a declaration for 'free'
       > formats/ods.c:211:35: error: call to undeclared function 'atof'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
       >                        double l = atof((char *) value);
       >                                   ^
       > 1 warning and 3 errors generated.
       > make: *** [<builtin>: formats/ods.o] Error 1
       For full logs, run 'nix-store -l /nix/store/pinl7arfp0w0d5kq48xdsvsz1la8d1v7-sc-im-0.8.3.drv'.
❯ pass githUB
Error: githUB is not in the password store.
❯ pass github.com
user: aaronpkelly, pass: PatrioticTowardRapidTent

Personal Access Token (for pushing commits over https, use in-place of your password): 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

Unfortunately, 2fa is mandatory: "Because of your contributions on GitHub, two-factor authentication will be required for your account starting Apr 27, 2023"

recovery codes (16 of them):
ae289-ede06 f1e09-cbd81 63d34-86841 2c12b-f62fe 6633e-5524a a6364-b5e50 9ae38-54661 7993e-7b9cb 3af47-53acd b7748-c6bbc cb034-0b3f7 eb781-d39b8 ad87c-fe1d6 10105-e2a59 d5c71-91718 6b95a-c94a2
```

### Additional context

I only tested on darwin 24.5, I haven't tested if the build is successful on any other archs

### Notify maintainers

@dotlambda

### Metadata

Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
❯ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-darwin"`
 - host os: `Darwin 19.6.0, macOS 10.15.7`
 - multi-user?: `yes`
 - sandbox: `no`
 - version: `nix-env (Nix) 2.18.2`
 - channels(root): `"nixpkgs-24.05-darwin"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixpkgs`
```

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/applications/misc/sc-im/default.nix
+++ b/pkgs/applications/misc/sc-im/default.nix
@@ -46,6 +46,8 @@ stdenv.mkDerivation rec {
 
   makeFlags = [ "prefix=${placeholder "out"}" ];
 
+  env.NIX_CFLAGS_COMPILE = lib.optionalString stdenv.cc.isClang "-Wno-error=implicit-function-declaration";
+
   postInstall = ''
     wrapProgram "$out/bin/sc-im" --prefix PATH : "${lib.makeBinPath [ gnuplot ]}"
   '';

